test/unit/server_tests.rb

```diff
@@ -172,9 +121,9 @@ class InitTests < UnitTests
       @error_procs = Factory.integer(3).times.map{ proc{} }
       @error_procs.each{ |p| @server_class.error(&p) }
 
-      @server_class.router do
-        service Factory.string, TestHandler.to_s
-      end
+      # @server_class.router do
+      #   service Factory.string, TestHandler.to_s
+      # end
```

Comment:
@jcredding Here is the test I removed in the server tests that previously required this: https://github.com/redding/sanford/pull/173/files#diff-5486fe40a1a7e9407522aa4c5c6ea6deL421